### PR TITLE
added parabolic region approach

### PR DESCRIPTION
**Pull request by nikita-astronaut**
Hello! I added parabolic region approach and tests to it. Seems like everything works.

However, in hyperbolic region, if we take their initial guess, it fails for high ecc.

So, it indeed works faster for small eccs, but for ecc like 3200, Newton can not converge with their initial guess. SO I decided not to add it, at least for now.

### CODE REVIEW

File: src/poliastro/twobody/angles.py [hunk 1]
```diff
@@ -24,6 +24,126 @@ def _kepler_equation_prime_hyper(F, M, ecc):
...
+    tolerance : float (optional)
+        smallness of the last term in series
+    Returns
+    -------
+    M_parabolic : float
+        kepler equation r.h.s.
+
+    Note
+    -----
+    Taken from the paper <<Robust resolution of Kepler’s equation in all eccentricity regimes>>,
```
Comment (Juanlu001) [inline src/poliastro/twobody/angles.py]: Could you change the citation style here to something more standard? For example (taken from scholar.google.com, Chicago style):

```
.. [1] Farnocchia, Davide, Davide Bracali Cioci, and Andrea Milani.
   "Robust resolution of Kepler’s equation in all eccentricity regimes."
   Celestial Mechanics and Dynamical Astronomy 116, no. 1 (2013): 21-34.
```

(as explained in https://numpydoc.readthedocs.io/en/latest/format.html#sections, section 13)

Comment (Juanlu001) [reply src/poliastro/twobody/angles.py]: Also, rename `Note` to `Notes` (and adjust the `-----`).

**Comment by nikita-astronaut** [comment]
Hello! Thanks for the review)

I added initial guess from the paper for hyperbolic region, but for non-high eccenricies. Weens like for ecc = 3200, their initial guess fails, but ours is robust, stable and works. But for smaller eccs, their initial guess works faster, so I added it. The threshold of 10 is chosen at random)

**Comment by Juanlu001** [comment]
Now there are some convergence failures though

**Comment by nikita-astronaut** [comment]
With their initial guess, there are always tests not converging. The problem is with large \Delta t and ecc, cause it was not included into the paper tests. So, our initial guess, as compared to the paper, is around 1 iteration slower, but at least stable.

**Comment by Juanlu001** [comment]
I think this is good to go!

**Pull request merged**